specification/cognitiveservices/data-plane/Language/stable/2022-05-01/conversationalanalysisauthoring.json

```diff
@@ -0,0 +1,3060 @@
+          }
+        }
+      },
+      "x-ms-client-name": "UtteranceEntitiesEvaluationResult"
+    },
+    "ConversationalAnalysisAuthoringUtteranceEntityEvaluationResult": {
+      "required": [
+        "category",
+        "offset",
+        "length"
+      ],
+      "type": "object",
+      "properties": {
+        "category": {
+          "description": "Represents the entity category.",
+          "type": "string"
+        },
+        "offset": {
+          "format": "int32",
+          "description": "Represents the entity offset index relative to the original text.",
+          "type": "integer"
+        },
+        "length": {
+          "format": "int32",
+          "description": "Represents the entity length.",
+          "type": "integer"
+        }
+      },
+      "x-ms-client-name": "UtteranceEntityEvaluationResult"
+    },
+    "ConversationalAnalysisAuthoringUtteranceEvaluationResult": {
+      "required": [
+        "text",
+        "language",
+        "entitiesResult",
+        "intentsResult"
+      ],
+      "type": "object",
+      "properties": {
+        "text": {
+          "description": "Represents the utterance text.",
+          "type": "string"
+        },
+        "language": {
+          "description": "Represents the utterance language.",
+          "type": "string"
+        },
+        "entitiesResult": {
+          "description": "Represents the entities results for the utterance.",
+          "$ref": "#/definitions/ConversationalAnalysisAuthoringUtteranceEntitiesEvaluationResult"
+        },
+        "intentsResult": {
+          "description": "Represents the intents results for the utterance.",
+          "$ref": "#/definitions/ConversationalAnalysisAuthoringUtteranceIntentsEvaluationResult"
+        }
+      },
+      "x-ms-client-name": "UtteranceEvaluationResult"
+    },
+    "ConversationalAnalysisAuthoringUtteranceIntentsEvaluationResult": {
+      "required": [
+        "expectedIntent",
+        "predictedIntent"
+      ],
+      "type": "object",
+      "properties": {
+        "expectedIntent": {
+          "description": "Represents the utterance's expected intent.",
+          "type": "string"
+        },
+        "predictedIntent": {
+          "description": "Represents the utterance's predicted intent.",
+          "type": "string"
+        }
+      },
+      "x-ms-client-name": "UtteranceIntentsEvaluationResult"
+    }
+  },
+  "parameters": {
+    "ConversationalAnalysisAuthoringFormatInQuery": {
+      "in": "query",
+      "name": "format",
+      "description": "The format of the project to export. The currently supported format is `clu`.",
+      "type": "string",
+      "enum": [
+        "clu",
+        "luis"
+      ],
+      "x-ms-enum": {
+        "name": "ExportedProjectFormat",
+        "modelAsString": true
+      },
+      "x-ms-parameter-location": "method"
+    },
+    "ConversationalAnalysisAuthoringStringIndexTypeInQuery": {
```

Comment:
Could you use the `StringIndexType` in common.json instead?

Reply:
The authoring still doesn't support all the StringIndexType that exists in the common.json file. That's why it's defined here with a subset of the values. We're planning to support the same set of values in the future. Then, we'll reference the one in common.json instead.